lib/box-node-sdk.js

```diff
@@ -118,6 +118,67 @@ BoxSDKNode.collaborationRoles = BoxSDKNode.prototype.collaborationRoles = BoxCli
 BoxSDKNode.CURRENT_USER_ID = BoxSDKNode.prototype.CURRENT_USER_ID = BoxClient.prototype.CURRENT_USER_ID;
 
 /**
+ * Gets the BoxSDKNode instance by passing boxAppSettings json downloaded from the developer console.
+ *
+ * @param {object} appConfig boxAppSettings json is passed as parameter.
+ * @returns {BoxSDKNode} the BoxSDKNode instance
```

Comment:
It might be good to describe the return value better; WHICH instance?  It's an instance that has been preconfigured with the values from the Dev Console